lib/puppet_metadata/beaker.rb

```diff
@@ -36,16 +36,19 @@ class << self
       #   limitations.
       #   When a boolean, it's applied on applicable operating systems. On arrays
       #   it's applied only when os is included in the provided array.
+      # @param [Optional[String]] domain
+      #   The desired domain (if use_fqdn is true)
```

Comment:
Could you extend the documentation for `use_fqdn` to state it's deprecated in favor of `domain`?
```suggestion
      #   Enforce a domain to be appended to the hostname, making it an FQDN
```